controllers/codeready_reconciler.go

```diff
@@ -26,14 +26,33 @@ import (
 	"sigs.k8s.io/yaml"
 )
 
+var codereadylabels = map[string]string{
+	"app.kubernetes.io/part-of": "codeready",
+}
+
+const (
+	CODEREADYNAMESPACENAME          = "workspaces"
+	CODEREADYOPERATORGROUPNAME      = "codeready-workspaces"
+	CODEREADYREDHATSUBSCRIPTIONNAME = "codeready-workspaces"
+	CODEREADYPACKAGENAME            = "codeready-workspaces"
+	CODEREADYDEPLOYMENTNAME         = "codeready-operator"
+	CODEREADYCUSTOMRESOURCENAME     = "codereadyworkspaces"
+	CODEREADYDEPLOYMENTSTATUSNAME   = "codeready"
```

Comment:
@MaazMS change this to CODEREADYDEPLOYMENTNAME

Reply:
@rasheedamir should i  change CODEREADYDEPLOYMENTNAME to CODEREADYDEPLOYMENTSTATUS